src/jit/importer.cpp

```diff
@@ -2878,24 +2878,36 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
                                        CORINFO_SIG_INFO *       sig,
                                        int                      memberRef,
                                        bool                     readonlyCall,
-                                       CorInfoIntrinsics *      pIntrinsicID)
+                                       CorInfoIntrinsics *      pIntrinsicID,
+                                       bool *                   pMustExpand)
 {
-    CorInfoIntrinsics intrinsicID = info.compCompHnd->getIntrinsicID(method);
+    CorInfoIntrinsics rawIntrinsicID = info.compCompHnd->getIntrinsicID(method);
```

Comment:
That's right. I overlooked getIntrinsicID is invoked everywhere. 
How hard would it to add optional argument to the existing JIT/EE interface? Should i go thru all kinds of JIT/EE implementation including desktop?